app/services/search/fetch-return-log-search-results.service.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+/**
+ * Handles fetching search results for return logs on the /search page
+ * @module FetchReturnLogSearchResultsService
+ */
+
+const { ref } = require('objection')
+
+const DatabaseConfig = require('../../../config/database.config.js')
+const ReturnLogModel = require('../../models/return-log.model.js')
+
+const RETURN_REFERENCE_PATTERN = /^\d+$/
```

Comment:
We like constants like this. No, it's not reused. But it allows us to put a name to the 'thing' when it is used, making the code more readable.